src/rez/pip.py

```diff
@@ -202,6 +202,10 @@ def find_pip_from_context(python_version, pip_version=None):
 
     py_exe = find_python_in_context(context)
 
+    if not py_exe:
+        print_debug("Failed to locate python executable from context")
```

Comment:
This log might be redundant I think. The `_check_found` function already logs when something isn't found and I also have a personal preference to let the caller log instead of letting the callee (find_pip_from_context) log.

Reply:
Alright, to me the log seemed consistent with code above and also acted as a comment for the `return` statement.

But I am fine with removing it.

Fixed in [2a64909](https://github.com/nerdvegas/rez/pull/1236/commits/2a64909caa1044fe871c2efbfe1b6443d61cfd61)